ssx-server.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:bullseye
```

Comment:
I don't have much experience dockerising javascript projects, any particular reason for not using the `node` base image, and alpine or a smaller image?

Reply:
Oh I was just curious, no need to change it